packages/hop-node/src/watchers/SettleBondedWithdrawalWatcher.ts

```diff
@@ -116,6 +111,15 @@ class SettleBondedWithdrawalWatcher extends BaseWatcher {
       }
 
       for (const bonder of bonderSet.values()) {
+        // if bonder has custom routes configured, then it should only check it's own
+        // bonded withdrawals to see if they need settlement
+        if (this.hasCustomRoutes) {
```

Comment:
Given our new understanding, I believe all bonders should have custom routes, so the concept of `hasCustomRoutes` can be removed. If a bonder wants all routes enabled (the `availableLiquidity` worker, for example), then they can just explicitly state every route in the config.

Reply:
Removed in https://github.com/hop-protocol/hop/commit/4d5b5a96e8c1b000bda20ca710aaeaaab5ce568e

Going forward, should the bonder config be explicitly set for other things, like fee bps, enabled networks, rpc urls, etc? basically have everything set to off/0 values unless explicitly set

Reply:
> basically have everything set to off/0 values unless explicitly set

I think that makes a ton of sense now that we have routes. Only consideration is that we wouldn't want to accidentally default to `0` if not set and then accept, for example, fees of `0`. Maybe upon startup there is an "if route check that values are non-`0`" or something like that, but I think, in general, this makes a ton of sense.